tools/cldr-code/src/main/java/org/unicode/cldr/util/CLDRFileOverride.java

```diff
@@ -33,7 +35,12 @@ public XMLSourceMapOverride(XMLSource source, Map<String, String> overrides) {
                     .forEach(
                             x -> {
                                 if (source.getValueAtDPath(x) == null) {
-                                    throw new IllegalArgumentException();
+                                    System.out.println(
+                                            "source.getValueAtDPath null in XMLSourceMapOverride");
```

Comment:
This happens when running the server and going to http://localhost:9080/cldr-apps/v#/am/SpeedAcceleration/6e5a3d520195cbe2

Mysteriously it doesn't happen when running tests in spite of the new data in testLightSpeed

Reply:
The plot thickens: the failure occurs for locale en, when loading the page for locale am!

Reply:
The caller is:

```
        public String getDisplayExample() {
            String displayExample = null;
            if (displayName != null) {
                displayExample =
                        sm.getComparisonValuesExample().getNonTrivialExampleHtml(xpath, rawEnglish);
            }
            return displayExample;
        }
```